modules/apps/portal-search/portal-search-admin-web/src/main/java/com/liferay/portal/search/admin/web/internal/portlet/action/EditMVCActionCommand.java

```diff
@@ -219,15 +214,16 @@ protected void reindexIndexReindexer(ActionRequest actionRequest)
 
 		IndexReindexer indexReindexer = _indexReindexers.get(className);
 
-		indexReindexer.reindex(_portalInstancesLocalService.getCompanyIds());
+		indexReindexer.reindex(
```

Comment:
Hey Guys, @oliv-yu @thektan 

Please don't forget this criteria from the story:
>The backend should perform validation on the provided company IDs to prevent submitting invalid instance IDs.